openedx/core/djangoapps/user_api/partition_schemes.py

```diff
@@ -10,6 +10,27 @@
 log = logging.getLogger(__name__)
 
 
+class NotImplementedPartitionScheme(object):
```

Comment:
Overzealous copy-pasting of the below class, I've removed it now. Thanks for the catch!